### ISSUE TITLE
Adding prelamin A antibody

### ISSUE THREAD

**Issue by Fabien-Maury** [obophenotype/human-phenotype-ontology#10397]
**Preferred term label:**
lamin A antibody positivity

**Synonyms**
prelamin A antibody positivity|PCNA antibody positivity|ki-67 antibody positivity|cyclin antibody positivity|LMNA antibody positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against human lamin A.
(PMID:1959878|PMID:2271902)

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:536|ORPHA:2137

**Your nano-attribution (ORCID)**
0000-0003-2854-0167

**Comment by pnrobinson** [comment]
@Fabien-Maury 
Adding new term: 
- ID: HP:6000643
- label: Anti-lamin A antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:1959878|PMID:2271902
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against lamin A.
- comment: 
- orcid: orcid:0000-0003-2854-0167
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10397
- synonyms: ki-67 antibody positivity(NONE)|LMNA antibody positivity(NONE)|PCNA antibody positivity(NONE)|Prelamin A antibody positivity(NONE)
